playbooks/roles/common-server-init/templates/update-chkrootkit-log

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+#
+# This script updates the expected log daily so each change is notified only
+# once.
+#
+
+LOG_DIR=/var/log/chkrootkit
+
+cp -a -f $LOG_DIR/log.today $LOG_DIR/log.expected
```

Comment:
@viadanna, it might be a good idea to retain the version before the current version of the expected file.

Reply:
:+1: